docs/guide/index.asciidoc

```diff
@@ -68,6 +68,20 @@ open source Kibana dashboards that come with APM Server.
 We designed Elastic APM to run on anything from a regular laptop to thousands of machines,
 and it's easy to get started.
 
+[[separate-component]]
+== Why is APM Server a separate component?
+
+The APM Server is kept as a separate component for the following reasons:
+
+* Agents can offload their heavy lifting tasks to the APM Server which can be scaled independently,
+  since it is a separate component thereby reducing the load on your app and by keeping the agents as light as possible.
+* During real-user-monitoring when the data is being collected in the browsers agents talk to the independent APM Server which acts as a central point to control the amount of data flowing into Elasticsearch and also control the pace of ingesting data.
```

Comment:
`During real-user-monitoring` > `For real-user-monitoring, data is being collected in agents running in browsers. Browsers communicate with the APM Server which acts...`